api/lib/ZwaveClient.ts

```diff
@@ -3929,6 +3930,24 @@ class ZwaveClient extends TypedEventEmitter<ZwaveClientEventCallbacks> {
 		throw new DriverNotReadyError()
 	}
 
+	/**
+	 * Used to trigger an update of controller FW via the Z-Wave JS firmware update service
+	 */
+	async firmwareAutoUpdateOTW(
```

Comment:
I don't really like this name. Suggestions?

Reply:
controllerFwUpdateOTW ?

Reply:
The other method (below) is called `firmwareUpdateOTW`, which also updates the controller.
Maybe they should just be merged and accept both a `FwFile` and a `FirmwareUpdateInfo`? Then decide based on the argument what to do.

Reply:
yep I think that would be better